app/views/shared/_alerts.html.erb

```diff
@@ -1,8 +1,8 @@
-<% flash.to_hash.slice('alert', 'info', 'notice', 'error').each do |name, msg| %>
-  <div class="global-alert global-alert--<%= name %>">
-    <div class="global-alert__content-container">
-      <strong class="global-alert__heading"><%= name.titleize %></strong>
-      <%= content_tag :p, msg.html_safe, class: 'global-alert__message t-flash-message' %>
+<% flash.to_hash.slice('notice', 'success', 'info', 'alert', 'error').each do |name, msg| %>
+  <div class="rad-notification rad-notification--<%= name %>">
```

Comment:
Can the notification markup be moved into a helper or something? Feels like there's a bit of duplication going on.